auth/oidc.go

```diff
@@ -680,7 +679,7 @@ func getExpiration(header http.Header) (ttl time.Duration, ok bool, err error) {
 		return 0, false, nil
 	}
 	expires := header.Get("Expires")
-	if expires == "" {
+	if expires == "" || expires == "0" {
```

Comment:
This makes sense, but should fetchCustomProviderConfig also just be setting ttl=MaxProviderConfigSyncInterval if cacheable() returns error, instead of failing to initialize the metadata?

Reply:
Yeah, we need to set ttl=MaxProviderConfigSyncInterval if cacheable() returns error, instead of failing to initialize the metadata. Right now we are just logging the error at debug level. Do you think, logging a warning would be appropriate here?

Reply:
Thank you Adam, As discussed this morning revised this to log info level in case of any cachable error.